scratchapi2/user.py

```diff
@@ -97,35 +121,34 @@ def studios(self, limit=3, offset=0):
                 created_at=studio["history"]["created"],
                 last_modified=studio["history"]["modified"],
                 followers=studio["stats"]["followers"]
-                ))
-        return list_studios
+            )
 
 class User(object):
-    """ Scratch User Class """
+    """Represents a Scratch user."""
+
+    username = None
+    
     def __init__(self, username, getinfo=True):
-        """ Set up user. requires one argument (username) at least """
+        """Initialize a User."""
         self.username = username
         if getinfo:
             self.info()
 
     def __str__(self):
-        """ Return string <User (username)> """
+        """Represent a User."""
         return "<User {0}>".format(self.username)
 
     __repr__ = __str__
 
     def __eq__(self, other):
-        """ Equal means that usernames are equal """
-        return self.username == other.username
+        """If the usernames are equal, the Users are considered equal.
+        (Equality is case-insensitive)

```

Comment:
I made a special note of this here because that's something that most Scratch users don't realize - usernames are not case-sensitive.